hms-api/src/handler/user/create.ts

```diff
@@ -1,21 +1,20 @@
 /* eslint-disable require-jsdoc */
 
-import {buildResponse} from '../../rest/responses';
-import {createUser} from '../../service/user-service';
-import {wrapHandler} from '../handler-wrapper';
-import UserCreateRequest from '../../rest/UserCreateRequest';
-import UserCreateResponse from '../../rest/UserCreateResponse';
+import { buildResponse } from "../../rest/responses";
+import { createUser } from "../../service/user-service";
+import { wrapHandler } from "../handler-wrapper";
+import UserCreateRequest from "../../rest/UserCreateRequest";
+import UserCreateResponse from "../../rest/UserCreateResponse";
```

Comment:
single quotes is right for the backend, and no space between the import and the `{` or `}`

Reply:
goes for all the other files too